src/licensedcode/models.py

```diff
@@ -772,6 +773,81 @@ def get_rules(
     return chain(licenses_as_rules, rules)
 
 
+def get_license_dirs(
+    additional_dirs,
+):
+    """
+    Takes in a list of additional directories specified during license detection
+    and produces a list of all the subdirectories containing license files.
+    """
+    # convert to absolute path in case user passes in a relative path, which messes up building rules from licenses
+    return [f"{str(Path(path).absolute())}/licenses" for path in additional_dirs]
+
+
+def get_rule_dirs(
+    additional_dirs,
+):
+    """
+    Takes in a list of additional directories specified during license detection
+    and produces a list of all the subdirectories containing rule files.
+    """
+    return [f"{str(Path(path).absolute())}/rules" for path in additional_dirs]
+
+
+def get_paths_to_installed_licenses():
+    from importlib_metadata import entry_points
+    from plugincode.location_provider import get_location
+    installed_plugins = entry_points(group='scancode_location_provider')
+    paths = []
+    for plugin in installed_plugins:
+        if plugin.name.startswith('licenses'):
```

Comment:
I think it would be better to extract the custom license plugin prefix (`'licenses'`) to a variable after line 39, something along the lines of `EXTERNAL_LICENSE_PLUGIN_PREFIX = 'licenses'`, then use that variable here. It would make it easier to see and change what the external license plugin prefix is.